libraries/Servo/src/stm32/Servo.cpp

```diff
@@ -29,7 +29,7 @@ static volatile int8_t timerChannel[_Nbr_16timers] = {-1}; // counter for the se
 
 static HardwareTimer TimerServo(TIMER_SERVO);
 
-uint8_t ServoCount = 0;                                    // the total number of attached servos
+static uint8_t ServoCount = 0;                                    // the total number of attached servos
```

Comment:
From my point of view there is no restriction to define this variable static.
Nevertheless, if this cause you issue in Marlin, this probably means you have a variable with the same name ? Can you confirm ? maybe you also need to take some action on marlin to avoid conflict too?

Reply:
Here is link to the marlin change I made https://github.com/MarlinFirmware/Marlin/pull/16151#pullrequestreview-328631174

Somehow this made it work, if I turn it back to normal it stops working.